ArduCopter/Parameters.cpp

```diff
@@ -1000,7 +1000,7 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @Param: FLIGHT_OPTIONS
     // @DisplayName: Flight mode options
     // @Description: Flight mode specific options
-    // @Bitmask: 0:Disable thrust loss check, 1:Disable yaw imbalance warning, 2:Release gripper on thrust loss, 3:Require position for arming
+    // @Bitmask: 0:Disable thrust loss check, 1:Disable yaw imbalance warning, 2:Release gripper on thrust loss (deprecated), 3:Require position for arming
```

Comment:
```suggestion
    // @Bitmask: 0:Disable thrust loss check, 1:Disable yaw imbalance warning, 2:Release gripper on thrust loss (removed), 3:Require position for arming
```

If it were deprecated, it would still work but issue a warning.

Reply:
ok, changed to (removed in 4.6+).  By the way there are other places (like in the enum and warning to user) which still uses the word "deprecated" but I've left these.  I think it's clear to the user what the issue is despite being possibly slightly incorrect wording.